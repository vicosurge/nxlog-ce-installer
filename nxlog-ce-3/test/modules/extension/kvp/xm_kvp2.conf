include tmp/common.conf
<Extension kvp>
    Module	xm_kvp
    KVPDelimiter ;
    KVDelimiter =
    EscapeChar \\
    EscapeControl TRUE
</Extension>

<Extension json>
    Module	xm_json
</Extension>

<Input in>
    Module	im_file
    File	"modules/extension/kvp/xm_kvp2.in"
    SavePos	FALSE
    ReadFromLast FALSE
    Exec	if $raw_event =~ /^#/ drop(); \
                else \
                { \
                   kvp->parse_kvp(); delete($EventReceivedTime); to_json(); kvp->reset_kvp(); \
                }
</Input>

<Output out>
    Module	om_file
    File	'tmp/output'
</Output>

<Route 1>
    Path	in => out
</Route>