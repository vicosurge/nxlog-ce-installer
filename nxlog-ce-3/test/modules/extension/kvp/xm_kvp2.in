key1='\v\\a\'lue1\'';key2='value2'
key1='\'\r\n';key2='value2'
'\'key1\''='value1';'\\key2'='value2'
key1=value1;key2=;key3=value3;
