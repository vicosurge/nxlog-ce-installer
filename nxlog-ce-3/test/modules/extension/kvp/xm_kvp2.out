{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"\\v\\a'lue1'","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"'\r\n","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","'key1'":"value1","\\key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":null,"key3":"value3"}
